tools/pipelines/build-docs-website-deploy.yml

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+# Licensed under the MIT License.
+
+# build-docs-website-deploy pipeline
+
+name: $(Build.BuildId)
+
+schedules:
+  - cron: "0 4 * * *"
+    displayName: Daily 8pm (PST) website deploy
+    branches:
+      include:
+        - main
+    always: true
+
+trigger: none
+pr: none
+
+parameters:
+- name: deployOverride
+  displayName: Deployment Override (default = based on branch)
+  type: string
+  default: default
+  values:
+    - default
+    - skip
+    - force
+- name: deployEnvironment
+  displayName: Static web app environment to deploy to
+  type: string
+  default: new
+  values:
+    - new
+    - old
+
+variables:
+- name: shouldDeploy
+  value: ${{ or(
+    eq(parameters.deployOverride, 'force'),
+    and(eq(variables['Build.SourceBranchName'], 'main'), eq(parameters.deployOverride, 'default'))
+    )}}
+- name: deploymentToken
+  ${{ if eq( parameters['deployEnvironment'], 'new' ) }}:
+    value: "$(FLUID_WEBSITE_TORUS_API_TOKEN)"
+  ${{ if eq( parameters['deployEnvironment'], 'old') }}:
+    value: "$(AZURE_STATIC_WEB_APPS_API_TOKEN)"
```

Comment:
These two variables come from the `storage-vars` variable group so we need to bring that one to this template. `build-docs` will also want to keep it for STORAGE_ACCOUNT and STORAGE_KEY though.